packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBitcoinAddresses/index.js

```diff
@@ -8,7 +8,7 @@ import SelectBoxBitcoin from './template'
 
 class SelectBoxBitcoinAddresses extends React.PureComponent {
   getLabel (coin) {
-    return `All Bitcoin${coin === 'BCH' ? ' Cash' : ''} Wallets`
+    return this.props.optional ? 'N/A' : `All Bitcoin${coin === 'BCH' ? ' Cash' : ''} Wallets`
```

Comment:
Why would you want to display 'N/A'

Reply:
For importing external btc addrs, there needs to be an option to deselect if you've made a selection.